website/docs/streaming-lakehouse/integrate-data-lakes/paimon.md

```diff
@@ -92,14 +96,29 @@ For further information, refer to Paimon’s [SQL Query documentation](https://p
 
 #### Union Read of Data in Fluss and Paimon
 
+##### Prerequisites
+Download the [fluss-lake-paimon-$FLUSS_VERSION$.jar](https://repo1.maven.org/maven2/org/apache/fluss/fluss-lake-paimon/$FLUSS_VERSION$/fluss-lake-paimon-$FLUSS_VERSION$.jar), and place it into `${FLINK_HOME}/lib`.
+
+##### Union Read
 To read the full dataset, which includes both Fluss (fresh) and Paimon (historical) data, simply query the table without any suffix. The following example illustrates this:
 
 ```sql title="Flink SQL"
 -- Query will union data from Fluss and Paimon
 SELECT SUM(order_count) AS total_orders FROM ads_nation_purchase_power;
 ```
+It can run both in batch and streaming mode, with Paimon as historical data and Fluss as fresh data:
```

Comment:
It supports both batch and streaming modes, using Paimon for historical data and Fluss for fresh data: